lib/internal/errors.js

```diff
@@ -114,6 +114,54 @@ function message(key, args) {
   return String(fmt.apply(null, args));
 }
 
+const primitiveTypes = [
+  'string', 'boolean', 'number', 'object', 'undefined', 'null',
+  'object', 'symbol', 'false', 'falsy', 'function', 'uint32'
+];
+function checkExpectedTypeInitial(expectedType) {
+  if (typeof expectedType !== 'string' && !Array.isArray(expectedType)) {
+    throw new exports.TypeError('ERR_INVALID_ARG_TYPE', 'expectedType',
+                                ['string', 'Array']);
+  }
+
+  if (typeof expectedType === 'string') {
+    const lowercase = firstLowerCase(expectedType);
```

Comment:
Since `primitiveTypes` is full of lowercase strings we can just do `expectedType.toLowerCase()`